<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDDStudentsInfoSetupDialog</class>
 <widget class="QDialog" name="ReportDDStudentsInfoSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>446</width>
     <height>443</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="3">
     <widget class="QCheckBox" name="chkPodtver">
      <property name="text">
       <string>Подтверждение оп&amp;латы</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QLabel" name="lblEndDatePodtver">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Окончание периода подтверждени&amp;я:</string>
      </property>
      <property name="buddy">
       <cstring>edtEndDatePodtver</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="4" colspan="3">
     <widget class="CDateEdit" name="edtEndDatePodtver">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="lblBegDatePodtver">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Начало пер&amp;иода подтверждения:</string>
      </property>
      <property name="buddy">
       <cstring>edtBegDatePodtver</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="3">
     <widget class="CDateEdit" name="edtBegDatePodtver">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblAge">
      <property name="text">
       <string>Возраст с</string>
      </property>
      <property name="buddy">
       <cstring>edtAgeFrom</cstring>
      </property>
     </widget>
    </item>
    <item row="10" column="3" colspan="4">
     <widget class="CPersonComboBoxEx" name="cmbPerson">
      <item>
       <property name="text">
        <string>Врач</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="3" colspan="4">
     <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
    </item>
    <item row="8" column="4" colspan="3">
     <widget class="CRBComboBox" name="cmbRefuseType">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="lblEndDate">
      <property name="text">
       <string>Дата окончания периода</string>
      </property>
      <property name="buddy">
       <cstring>edtEndDate</cstring>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="lblPerson">
      <property name="text">
       <string>&amp;Врач</string>
      </property>
      <property name="buddy">
       <cstring>cmbPerson</cstring>
      </property>
     </widget>
    </item>
    <item row="11" column="3" colspan="3">
     <widget class="COrgComboBox" name="cmbWorkOrganisation">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblPodtver">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Тип подтве&amp;рждения:</string>
      </property>
      <property name="buddy">
       <cstring>cmbPodtver</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblSex">
      <property name="text">
       <string>Пол</string>
      </property>
      <property name="buddy">
       <cstring>cmbSex</cstring>
      </property>
     </widget>
    </item>
    <item row="11" column="6">
     <widget class="QToolButton" name="btnSelectWorkOrganisation">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="12" column="5">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="3" colspan="3">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="4">
     <widget class="QFrame" name="frmAge_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="edtAgeFrom">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblAgeTo">
         <property name="text">
          <string>по</string>
         </property>
         <property name="buddy">
          <cstring>edtAgeTo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="edtAgeTo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblAgeYears">
         <property name="text">
          <string>лет</string>
         </property>
         <property name="buddy">
          <cstring>edtAgeTo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="lblRefuseType">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>При&amp;чина отказа:</string>
      </property>
      <property name="buddy">
       <cstring>cmbRefuseType</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="4">
     <widget class="QComboBox" name="cmbSex">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>М</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ж</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3" colspan="4">
     <widget class="CDateEdit" name="edtEndDate"/>
    </item>
    <item row="0" column="3" colspan="4">
     <widget class="CDateEdit" name="edtBegDate"/>
    </item>
    <item row="5" column="4" colspan="3">
     <widget class="QComboBox" name="cmbPodtver">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLabel" name="lblWorkOrganisation">
      <property name="text">
       <string>Учебное заведение</string>
      </property>
      <property name="buddy">
       <cstring>cmbWorkOrganisation</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="lblBegDate">
      <property name="text">
       <string>Дата начала периода</string>
      </property>
      <property name="buddy">
       <cstring>edtBegDate</cstring>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QLabel" name="lblOrgStructure">
      <property name="text">
       <string>&amp;Подразделение</string>
      </property>
      <property name="buddy">
       <cstring>cmbOrgStructure</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>COrgComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>chkPodtver</tabstop>
  <tabstop>cmbPodtver</tabstop>
  <tabstop>edtBegDatePodtver</tabstop>
  <tabstop>edtEndDatePodtver</tabstop>
  <tabstop>cmbRefuseType</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbWorkOrganisation</tabstop>
  <tabstop>btnSelectWorkOrganisation</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportDDStudentsInfoSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportDDStudentsInfoSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
